src/software/Localization/main_rigLocalizer.cpp

```diff
@@ -249,24 +253,30 @@ int main(int argc, char** argv)
 #endif
                                            );
     param = new localization::VoctreeLocalizer::Parameters();
```

Comment:
@fabiencastan for `param` i don't know, if I use unique_ptr i guess i have to change all the signatures that use the parameters. But it could be nice to start cleaning up this mess that i made
https://github.com/poparteu/openMVG/blob/popart_offlineLocalization/src/openMVG/localization/VoctreeLocalizer.cpp#L156